Django 5: Failing test test_admin_add

Following test will fail with Django >= 5.X
```
======================================================================
FAIL: test_admin_add (article.tests.AdminArticleTestCase.test_admin_add)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/source/example/article/tests.py", line 141, in test_admin_add
    self.assertInContent("<h1>Ajout de Article (Hollandais)</h1>", resp)
  File "/build/source/example/article/tests.py", line 40, in assertInContent
    return super().assertIn(member, smart_str(resp.content), msg)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: '<h1>Ajout de Article (Hollandais)</h1>' not found in '<!DOCTYPE html>\n\n<html lang="fr" dir="ltr">\n<head>\n<title>Ajout de Article (Néerlandais) | Site d’administration de Django</title>\n<link rel="stylesheet" href="admin/css/base.css">\n\n  <link rel="stylesheet" href="admin/css/dark_mode.css">\n 
[...]
```

--- a/example/article/tests.py
+++ b/example/article/tests.py
@@ -137,7 +137,9 @@ class AdminArticleTestCase(TestMixin, TestCase):
         resp = self.client.get(reverse("admin:article_article_add"))
         self.assertEqual(200, resp.status_code)
 
-        if django.VERSION >= (3, 0):
+        if django.VERSION >= (5, 0):
+            self.assertInContent("<h1>Ajout de Article (Néerlandais)</h1>", resp)
+        elif django.VERSION >= (3, 0):
             self.assertInContent("<h1>Ajout de Article (Hollandais)</h1>", resp)
         else:
             self.assertInContent("<h1>Ajout Article (Hollandais)</h1>", resp)

